src/shared/components/note/style.css

```diff
@@ -0,0 +1,18 @@
+@value mediaSmall from "../variables/breakpoints.css";
+@value red from "../variables/colors.css";
+
+.note {
+  background-color: #fffcd7;
+  padding: 20px 20px 5px 20px;
+  box-shadow: 1px 1px 2px 2px rgba(0,0,0,0.2);
+  border: 1px solid #d0d0d0;
+  width: 30.5%;
```

Comment:
I think note should not declare its width. This should be left to its parent component. This will be made easier with the updates to the Grid component.